<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Packing Slip Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMain">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_companyData">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="text">
          <string>Import Company Data Before Generating Label (File -&gt; Import Company Data)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_customer">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>SHIP TO:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>SOLD TO:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_orderID">
           <property name="placeholderText">
            <string>Order Number(s)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_cityStateZip">
           <item>
            <widget class="QLineEdit" name="lineEdit_cCity">
             <property name="placeholderText">
              <string>Cityville</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_cState">
             <property name="placeholderText">
              <string>WA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_cZip">
             <property name="placeholderText">
              <string>12345</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Purchase Order(s):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineEdit_cNumber">
           <property name="placeholderText">
            <string>555-555-5555</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_cName">
           <property name="placeholderText">
            <string>John Doe</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_cCountry">
           <property name="placeholderText">
            <string>United States</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_cStreet">
           <property name="placeholderText">
            <string>1234 Streetname Blvd</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QDateEdit" name="dateEdit">
           <property name="displayFormat">
            <string>MMMM dd, yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::TimeSpec::LocalTime</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>767</width>
            <height>194</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_orderData">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>799</width>
             <height>181</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="text">
            <string>Order List
Example Data Until Complete
QTY         ITEM
1           itemOne

            item attribute</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabItemEditor">
     <attribute name="title">
      <string>Item Editor</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Company_Data"/>
    <addaction name="actionExport_Company_Data"/>
    <addaction name="actionPrint_Packing_Slip"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionImport_Company_Data">
   <property name="text">
    <string>Import Company Data</string>
   </property>
  </action>
  <action name="actionExport_Company_Data">
   <property name="text">
    <string>Generate Company Data</string>
   </property>
  </action>
  <action name="actionPrint_Packing_Slip">
   <property name="text">
    <string>Print Packing Slip</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_cName</tabstop>
  <tabstop>lineEdit_cNumber</tabstop>
  <tabstop>lineEdit_cStreet</tabstop>
  <tabstop>lineEdit_cCity</tabstop>
  <tabstop>lineEdit_cState</tabstop>
  <tabstop>lineEdit_cZip</tabstop>
  <tabstop>lineEdit_cCountry</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEdit_orderID</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
